third-party/hwloc/Makefile

```diff
@@ -22,8 +22,7 @@ CHPL_HWLOC_CFG_OPTIONS += --enable-static \
                           --disable-cuda \
                           --disable-nvml \
                           --disable-opencl \
-                          --disable-gl \
-                          --disable-pci
```

Comment:
Mild concern about portability of enabling libpci. Do you think we should make this EX only to start?

Reply:
I've modified the Makefile to only enable libpci on the hpe-cray-ex platform.

Reply:
On second thought, we should make this based on comm=ofi instead of platform=ex